superset/models/sql_lab.py

```diff
@@ -86,7 +86,7 @@ class Query(
     user_id = Column(Integer, ForeignKey("ab_user.id"), nullable=True)
     status = Column(String(16), default=QueryStatus.PENDING)
     tab_name = Column(String(256))
-    sql_editor_id = Column(String(256))
+    sql_editor_id = Column(String(256), index=True)
```

Comment:
Given that it's now a index on two columns you'll have to use the [`Index`](https://docs.sqlalchemy.org/en/20/core/constraints.html#indexes) function.